cluster/ubuntu/util.sh

```diff
@@ -306,10 +306,7 @@ function kube-up() {
   source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE:-config-default.sh}"
 
   # downloading tarball release
-  if [[ -d "${KUBE_ROOT}/cluster/ubuntu/binaries" ]]; then
-    rm -rf "${KUBE_ROOT}/cluster/ubuntu/binaries"
-  fi
-  "${KUBE_ROOT}/cluster/ubuntu/download-release.sh"
```

Comment:
This is intentional. We don't want to use the out-dated binaries when the user reset $KUBE_VERSION var, and it won't easy to trace the corresponding version of the binaries. Too much chaos.